### PR TITLE
Fixes some apt-get update flakiness

### PR DESCRIPTION
**Pull request by ehankland**

### CODE REVIEW

**Comment by ehankland** [comment]
test this please.

**Comment by voellm** [comment]
LGTM

File: perfkitbenchmarker/package_managers.py [hunk 1]
```diff
@@ -213,7 +213,9 @@ def Startup(self):
 
   def AptUpdate(self):
     """Updates the package lists on VMs using apt."""
-    self.RemoteCommand('sudo apt-get update')
+    # We don't want to fail if updating fails. The '-m' option
+    # lets us continue even when we can't locate an archive.
+    self.RemoteCommand('sudo apt-get update -m', ignore_failure=True)
```
Comment (cmccoy) [inline perfkitbenchmarker/package_managers.py]: Maybe `--ignore-missing` here, to make the intent clear in the logs?

**Comment by cmccoy** [comment]
@ehankland: can we remove https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/package_managers.py#L231 now that this is in?

**Comment by ehankland** [comment]
I don't think so - the error with Azure packages was this:
E: Unable to locate package libaio-dev
E: Package 'libaio1' has no installation candidate"
Which occurs when we apt-get install

Whereas this PR was meant to fix this error:
W: Failed to fetch http://http.debian.net/debian/dists/wheezy-backports/main/binary-amd64/Packages  Hash Sum mismatch
E: Some index files failed to download. They have been ignored, or old ones used instead.
Which occurs when we apt-get update

**Comment by cmccoy** [comment]
Ah, OK. Thanks. Wishful thinking :)

On Wed, May 6, 2015 at 9:52 AM, ehankland <notifications@github.com> wrote:

> I don't think so - the error with Azure packages was this:
> E: Unable to locate package libaio-dev
> E: Package 'libaio1' has no installation candidate"
> Which occurs when we apt-get install
>
> Whereas this PR was meant to fix this error:
> W: Failed to fetch
> http://http.debian.net/debian/dists/wheezy-backports/main/binary-amd64/Packages
> Hash Sum mismatch
> E: Some index files failed to download. They have been ignored, or old
> ones used instead.
> Which occurs when we apt-get update
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/250#issuecomment-99535726>
> .
>



-- 
Connor McCoy | Software Engineer | connormccoy@google.com | 206-876-1657

**Pull request merged**